Corrected error on capital letter

alfcanr opened the pull request:
Changed mode form Mode, so now it works.
Now the mode descriptive works!
Cheers.

unDocUMeantIt committed:
Create README.md

asalber committed:
Fixed merge conflict.

asalber committed:
Fixed distribution plots.

asalber committed:
Fixed probability distribution plots.

asalber committed:
Fixed mistakes.

AlfCano committed:
Update common_functions.js

The "split is not a function" error occurs when we call the split() method on a value that is not of type string. To solve the error, convert the value to a string using the toString() method before calling the split method or make sure to only call the split method on strings.

https://bobbyhadz.com/blog/javascript-typeerror-string-split-is-not-a-function#:~:text=The%20%22split%20is%20not%20a,the%20split%20method%20on%20strings.

tfry-git committed:
Fix header.add() statement.

This is a cherry-pick of https://github.com/AlfCano/rkTeaching/commit/ecb953c5f130e11dcb8a6a5c98c750ab40dba4df
Thanks to AlfCano!

asalber committed:
Fixed errors.

asalber committed:
Merge branch 'master' of github.com:rkward-community/rk.Teaching

asalber committed:
Rewrite the code of most procedures to use tidyverse.

asalber committed:
Use the a bootstrap css theme as default css theme and fix some minor errors.

asalber committed:
Changed the order of packages dependencies.

asalber committed:
Rewrite scatter plot code to use tidyverse.

asalber committed:
Updated plugin translation.

asalber committed:
Added test for the variance of one population and power analysis for the t-test.

alfcanr committed:
Update descriptive_statistics.js

Error in caps Mode for mode.

alfcanr committed:
Update coins_probability_space.js

Missing the last letter "n"  in the word  "results" at the assignation of function "tosscoin", and at the end to the assign function to the global environment from the local one.

tfry-git commented on inst/rkward/plugins/parametric_tests/anova.js:
```diff
@@ -120,76 +125,98 @@ function printout() {
   echo(i18n("Source of variation") + ' = anova.results[["ANOVA"]][["Effect"]], ');
-  echo(i18n("Degrees of freedom <br/> numerator") + ' = anova.results[["ANOVA"]][["DFn"]], ');
-  echo(i18n("Degrees of freedom <br/> denominator") + ' = anova.results[["ANOVA"]][["DFd"]], ');
+  echo(i18n("Num DF") + ' = anova.results[["ANOVA"]][["DFn"]], ');
```
Spell out numerator / denominator?

tfry-git commented on inst/rkward/plugins/parametric_tests/one_sample_t_test.xml:
```diff
@@ -44,14 +44,13 @@
         <p>Insert the numeric variable to test.</p>
-        <p>If you want different tests for the categories of one or several factors, check the box          
-          <b>Test by groups</b>          and select the factors to define the groups. If you select more than one factor you will get one test for every category combination of the factors.
+        <p>If you want different tests for the categories of one or several factors, check the box <b>Test by groups</b> and select the factors to define the groups. If you select more than one factor you will get one test for every category combination of the factors.
```
Use <label> (see above)

tfry-git commented on inst/rkward/plugins/parametric_tests/one_sample_variance_test.xml:
```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE rkplugin>
+<document>
+  <code file="one_sample_variance_test.js"/>
+  <help file="one_sample_variance_test.rkh"/>
```
See above: Consider removing while help page is empty.

tfry-git commented on inst/rkward/plugins/parametric_tests/test_t_power.xml:
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE rkplugin>
+<document>
+  <code file="test_t_power.js"/>
+  <help file="test_t_power.rkh"/>
```
See above.

tfry-git commented on inst/rkward/plugins/plots/linesplot.xml:
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE rkplugin>
+<document>
+	<code file="linesplot.js"/>
+	<help file="linesplot.rkh"/>
```
See above.

tfry-git commented on rkward_output.css:
```diff
@@ -0,0 +1,12066 @@
+@charset "UTF-8";
```
Is this file actually used?

tfry-git reviewed (commented):
That is a huge amount of work! I am a little overwhelmed, and I did little more than scanning through the changes. I cannot pretend to have read (or even tested) everything. If there are any particular areas that you would like me to take a closer look at, please let me know.

I left some minor nitpicks, here and there, which you may or may not want to address. Merge when you consider this ready!

tfry-git commented:
Oh, I suppose this was targeted at master, and so the actual diff is much smaller?

@unDocUMeantIt : Perhaps it would avoid confusion to drop the develop branch, and just work in master (and / or dedicated feature branches)?

alfcanr committed:
Update README.md

As another github user has pointed out, there were missing points at the address of the rkward-community/rk.teaching ,
Regards

alfcanr commented:
Thank you.
Best regards.
Alfonso.

El mié, 2 jul 2025 a la(s) 8:45 a.m., Thomas Friedrichsmeier (
***@***.***) escribió:

> Closed #4 <https://github.com/rkward-community/rk.Teaching/pull/4>.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rkward-community/rk.Teaching/pull/4#event-18433783250>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATHZPRBUCB2IC6A6EEJA2B33GPWBBAVCNFSM6AAAAAB76YCC4WVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJYGQZTGNZYGMZDKMA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
